### PR TITLE
fix(dependabot): upgrading aws-crt to bump axios version

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes
Upgrading aws-crt and cmake-js to keep axios ^1.6.0

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "@types/react": "^18.2.22",
     "@types/react-dom": "^18.2.7",
     "browserslist": "^4.16.15",
+    "cmake-js/axios": "^1.6.0",
```
Comment (reesscot) [inline package.json:77]: do we know what this is used for?

Comment (esauerbo) [reply package.json:77]: Not sure how to interpret this but `yarn why cmake-js` shows 
``` 
=> Found "cmake-js@7.2.1"
info Has been hoisted to "cmake-js"
info Reasons this module exists
   - "workspace-aggregator-43cb8e75-af3d-4e3b-9a57-f6c44f620bea" depends on it
   - Specified in "dependencies"
   - Hoisted from "_project_#cmake-js"
  ```

File: package.json [hunk 2]
```diff
@@ -122,5 +123,9 @@
     "turbo": "^1.2.8",
     "typescript": "^5.2.2",
     "wait-on": "^7.2.0"
+  },
+  "dependencies": {
+    "aws-crt": "^1.19.0",
+    "cmake-js": "^7.2.1"
   }
```
Comment (calebpollman) [inline package.json:130]: There should be no hard dependencies in the top level _package.json_

**Review by calebpollman — COMMENTED** [review]
Curious, have you tried removing `aws-crt` as a dependency?

**Comment by esauerbo** [comment]
> Curious, have you tried removing `aws-crt` as a dependency?

Just did, it was a devDependency for e2e but still built and I ran a few e2e tests and nothing broke @calebpollman

**Comment by calebpollman** [comment]
> > Curious, have you tried removing `aws-crt` as a dependency?
> 
> Just did, it was a devDependency for e2e but still built and I ran a few e2e tests and nothing broke @calebpollman

Nice! Yeah was thinking with the JS dropping so much of the aws SDK as a dep that we may no longer need it

**Review by calebpollman — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Pull request merged**